classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3480,11 +3532,15 @@ static function get_publishablekey() {
 		if ( self::using_legacy_keys() ) {
 			$publishablekey = pmpro_getOption( 'stripe_publishablekey' ); 
 		} else {
-			$publishablekey = pmpro_getOption( 'pmpro_gateway_environment' ) === 'live' ? pmpro_getOption( 'live_stripe_connect_publishablekey' ) : pmpro_getOption( 'test_stripe_connect_publishablekey' );
+			$publishablekey = pmpro_getOption( 'gateway_environment' ) === 'live' ? pmpro_getOption( 'live_stripe_connect_publishablekey' ) : pmpro_getOption( 'test_stripe_connect_publishablekey' );
 		}
 		return $publishablekey;
 	}
 
+	static function get_connect_user_id() {
```

Comment:
```suggestion
	/**
	 * Get the Stripe Connect User ID based on gateway environment.
	 *
	 * @since 2.6.0
	 *
	 * @return string The Stripe Connect User ID.
	 */
	static function get_connect_user_id() {
```